hudi-hive/src/main/java/org/apache/hudi/hive/HoodieHiveClient.java

```diff
@@ -632,24 +632,33 @@ public void close() {
     if (!lastCommitTimeSynced.isPresent()) {
       LOG.info("Last commit time synced is not known, listing all partitions in " + syncConfig.basePath + ",FS :" + fs);
       try {
-        return FSUtils.getAllPartitionPaths(fs, syncConfig.basePath, syncConfig.assumeDatePartitioning);
+
+        List<String> fsPartitions = FSUtils.getAllPartitionPaths(fs, syncConfig.basePath, syncConfig.assumeDatePartitioning);
+        List<String> tlPartitions = findPartitionsAfter("0");
```

Comment:
Is there a better way to get the only commit that may be present? lets try to avoid hardcoding `0`

Reply:
> Is there a better way to get the only commit that may be present? lets try to avoid hardcoding `0`

Thanks, let me think about it.

Reply:
My idea is extracting the public method to get all the instants after startTs.